plugins/jspsych-iat.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * jspsych-single-stim
+ * Kristin Diep 
+ *
+ * plugin for displaying a stimulus and getting a keyboard response
+ *
+ * documentation: docs.jspsych.org
+ *
+ **/
+
+
+ jsPsych.plugins["iat"] = (function() {
+
+  var plugin = {};
+
+  jsPsych.pluginAPI.registerPreload('iat', 'stimulus', 'image');

```

Comment:
If we are allowing HTML content for stimulus, this needs to change slightly. See the example in single-stim for the fourth parameter that needs to get added.